### ISSUE TITLE
Files.toPath() crash on older Android versions

### ISSUE THREAD

**Issue by h-cspu** [gsantner/markor#1894]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Mordor**. Don't worry; Markor and Mordor appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

[log.txt](https://github.com/gsantner/markor/files/9763201/log.txt)

### Steps to reproduce

1. 
2. 
3. 

### Information

Android version: 7.1.2
Device: 
App Version: 2.10.5

### Source

F-Droid

### Format / File type

Not specific

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment]
It's this line crashing, seemingly the try-catch does't catch this one. So simple if (androidversio) check guard is needed too.

```
app/src/main/java/net/gsantner/opoc/util/GsFileUtils.java
455:            if (!GsTextUtils.isNullOrEmpty(t = Files.probeContentType(file.toPath()))) {
```

**Comment by gsantner** [comment]
It's not fixed, but noted the reason for the issue and what todo

**Comment by h-cspu** [comment]
I closed the issue on purpose because I don't want to contribute anymore.

**Comment by gsantner** [comment]
Fixed in <https://github.com/gsantner/markor/commit/7a04cdfe1b6299c16e6298766ca5c9fbb1e31c42>

**Comment by gsantner** [comment]
@h-cspu   
OK, sorry to say, but I just told you about reality at the search question. I mean it's written there explicitly, and I tried to tell you before in the other comment.   
Just pushing out information, requests, and questions without looking checking minimally on your own neither helps you, any reader who comes back here, nor me who has to read this all and spend my free time to answer and guess what you want.

And all the fields are there for that reason, to help reporters to tell as much information as required to reproduce the issue, understand what it is about and if it's meant to be used this way. 

Basically empty issues are not helpful to whoever wants to improve or fix something. This is not about me or Markor at all, this is about how the flow of Open Source development accepting issues works.

![grafik](https://user-images.githubusercontent.com/6735650/195416988-b1597ce1-1dd4-4a13-82ca-300b5fc8d21a.png)
